src/Model/Page.php

```diff
@@ -306,7 +306,7 @@ public function getName()
      */
     public function setSlug($slug)
     {
-        $this->slug = self::slugify(trim($slug));
+        $this->slug = self::slugify(trim((string) $slug));
```

Comment:
I hope nothing breaks if we change a slug from `null` to `''`. BTW, when does this occur? Can't see why the slug of a page would be `null` or `''`

Reply:
trim(null) = '' AFAIK

Reply:
https://3v4l.org/71CT5 vs https://3v4l.org/ufpTl, not sure this is our problem?

Reply:
https://3v4l.org/3egP6